jobs/2016-11-28-Logo-for-Libreflip.md

```diff
@@ -23,6 +22,12 @@ On the machine the Symbol will be left of the written Title, on the website the
 If arranged so that the symbol is left of the text in the same line, and scaled so that the whole logo is 8cm high, no single detail can be smaller then 4mm
 Currently I am trying to secure funding through various sources. when successful, I will pay you 50-100€ for designing a great logo. If I do not secure any funding (unlikely) then this will be a gratis/free job.
 
+More about the project:
+I am in the process od building and designing an opensource pageturning bookscanner. Right now there is only one opensource bookscanner and it is the one from google, that is kind of hard to get to work and it only works for books that are relativly new. new books usually are already available in digitial form, so no need to scan them.
```

Comment:
- Typo: in the process "OF" building
- Word separation: an "open source page turning book scanner" ? :)
- Capitalize: "New books" usually are